layouts/shortcodes/blocks/cover.html

```diff
@@ -24,7 +24,7 @@
 }
 </style>
 {{ end }}
-<section id="{{ $blockID }}" class="row td-cover-block td-cover-block--height-{{ $height }} js-td-cover td-overlay td-overlay--dark -bg-{{ $col_id }}">
+<section id="{{ $blockID }}" class="row td-cover-block td-cover-block--height-{{ $height }}{{ if not .Site.Params.ui.navbar_translucent_over_cover_disable }} js-td-cover{{ end }} td-overlay td-overlay--dark -bg-{{ $col_id }}">
```

Comment:
While not strictly necessary, this change prevents the navbar-style-fiddling JS code from being triggered when translucency is disabled: 
https://github.com/google/docsy/blob/db8f7a7528ee0387728c07312118c3f749aab3d8/assets/js/base.js#L35-L40